AUTHORS

```diff
@@ -1,8 +1,12 @@
 Darron Froese <dfroese@salesforce.com>
 Eric Hole <eric.hole@gmail.com>
+Igor Vuk <parcijala@gmail.com>
 Joe Beda <joe.github@bedafamily.com>
 Joshua Carp <jm.carp@gmail.com>
+@Kris__Nova <kris@nivenly.com>
 Kris Nova <kris@nivenly.com>
+Marko Mudrinić <mudrinic.mare@gmail.com>
 Michael Hausenblas <michael.hausenblas@gmail.com>
 Paul Czarkowski <username.taken@gmail.com>
 Ted Wood <ted@mailchimp.com>
+Ted Wood <ted@xy0.org>
```

Comment:
OK... nevermind, sorry to bug you on this. The only thing I can do here is revert #101 which is pointless because as soon as I make another commit I'll just wind up back in here with two email addresses again 😢  Since it looks like you're in there twice anyway, I'll look to find alternative ways of generating the authors file and PR it if I find something.